### DISCUSSION TITLE
Is it fixed??

### DISCUSSION THREAD

**Discussion by Fortnitegod4200069996** [3kh0/ext-remover#105] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i also wanna know how i can go back to a different version on a school Chromebook

**Comment by LogA1Cyber** [comment, marked as answer]
Same, my school just made it so none of the hacks work.

**Reply by Fortnitegod4200069996** [reply]
its not the schools its the google chrome updated that made it so you cant use bookmarks on extensions by admin i think

**Reply by LogA1Cyber** [reply]
what version is your chromebook on?

**Reply by Meta0Music** [reply]
im having the same problem and mine is on Google_Drawcia.13606.442.0

**Reply by LogA1Cyber** [reply]
I meant the version number of Google Chrome

**Reply by LogA1Cyber** [reply]
> im having the same problem and mine is on Google_Drawcia.13606.442.0
I have no idea what that means

**Reply by synapt0** [reply]
> > im having the same problem and mine is on Google_Drawcia.13606.442.0
> > I have no idea what that means

go to chrome://version/ and then look at the top

**Reply by Meta0Music** [reply]
106.0.5249.112 (Official Build) (64-bit)
